web/src/features/charts/graphUtils.ts

```diff
@@ -112,6 +113,34 @@ export const getNextDatetime = (datetimes: Date[], currentDate: Date) => {
   return datetimes[index + 1];
 };
 
+export function determineUnit(
+  displayByEmissions: boolean,
+  currentZoneDetail: ZoneDetail,
+  mixMode: Mode,
+  timeAverage: TimeAverages
+) {
+  if (displayByEmissions) {
+    return getUnit(
+      formatCo2(getTotalEmissionsAvailable(currentZoneDetail, mixMode)) +
+        ' ' +
+        t('ofCO2eq')
+    );
```

Comment:
Won't the output of this one always be the unit you're giving it, in which case we could just return that?
```suggestion
    return  t('ofCO2eq')
```

Reply:
The translation here is `of CO₂eq` but the formatCo2 function adds the number of tons e.g. `6 kt` so it is `6 kt of CO₂eq`  and then the get unit removes the number so it ends up being `kt of CO₂eq` like the unit is on the tooltip.